tests/installation/bootloader_s390.pm

```diff
@@ -273,7 +273,9 @@ sub format_dasd {
         record_soft_failure('bsc#1151436');
         script_run('dasd_reload');
         assert_script_run('dmesg');
+        assert_script_run('export ZDEV_DEBUG=1');
         assert_script_run("bash -x /sbin/dasd_configure -f $dasd_path 0");
+        assert_script_run('for i in /proc/[0-9]* ; do echo ; cat $i/comm ; cat $i/stack ; done >> DASDBugInformation.log');
```

Comment:
if the previous assert_script_run, bash -x /sbin/dasd_configure failed, you won't reachthis line to give you the output you would expect to be helpful.

this line is only reached when dasd_configure passes.; otherwise, the fail_hook of the module is started

Reply:
Hmmm... The export command and these both commands have to be combined.